classifier_load_testing/create_parents.rb

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ruby
```

Comment:
Note that `create_parents` is `init` renamed.

Also, since we can get IDs from group names, I figured it didn't make sense to output the parent IDs anymore. So I just outputted if the groups were successfully created (and the group names).